<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeControls</class>
 <widget class="QWidget" name="TimeControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>start:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="start">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>d.M.yyyy h:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>stop:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="stop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>d.M.yyyy h:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>dt:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="dt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string>secs</string>
     </property>
     <property name="minimum">
      <number>60</number>
     </property>
     <property name="maximum">
      <number>86400</number>
     </property>
     <property name="singleStep">
      <number>60</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="apply_time_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
